### ISSUE TITLE
java.util.concurrent.TimeoutException: Process timed out.

### ISSUE THREAD

**Issue by grmoon** [GoogleCloudPlatform/cloud-code-intellij#2841]
**Platform:** WebStorm
**OS:** Mac OS X
**Plugin Version**: 2020.2.3

```
java.util.concurrent.TimeoutException: Process timed out.
	at com.google.cloud.tools.intellij.util.ProcessExtensionsKt$waitForProcessOutput$1.invokeSuspend(ProcessExtensions.kt:109)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:270)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:79)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:54)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:36)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at com.google.cloud.tools.intellij.util.ProcessExtensionsKt.waitForProcessOutput(ProcessExtensions.kt:93)
	at com.google.cloud.tools.intellij.login.sdk.CloudSdkAuthWrapper.executeGcloud(CloudSdkAuthWrapper.kt:211)
	at com.google.cloud.tools.intellij.login.sdk.CloudSdkAuthWrapper.executeGcloud$default(CloudSdkAuthWrapper.kt:199)
	at com.google.cloud.tools.intellij.login.sdk.CloudSdkAuthWrapper.getConfigHelper(CloudSdkAuthWrapper.kt:170)
	at com.google.cloud.tools.intellij.login.sdk.CloudSdkTokenExpiryScheduler.scheduleTokenExpiryUpdate(CloudSdkTokenExpiryScheduler.kt:47)
	at com.google.cloud.tools.intellij.login.sdk.CloudSdkLoginService$loadPersistedCredentials$1.invokeSuspend(CloudSdkLoginService.kt:131)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:740)
```

**Comment by grmoon** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/2965)

**Comment by ivanporty** [comment]
Fixed in 20.12.2
